image_classification/inception_v4.py

```diff
@@ -0,0 +1,526 @@
+        num_channels=1536,
+        num_filters=384,
+        filter_size=1,
+        stride=1,
+        padding=0)
+    b2_conv1 = img_conv(
+        name='inceptC{0}_branch2_conv1'.format(depth),
+        input=b2_conv0,
+        num_channels=384,
+        num_filters=256,
+        filter_size=(1, 3),
+        stride=1,
+        padding=(0, 1))
+    b2_conv2 = img_conv(
+        name='inceptC{0}_branch2_conv2'.format(depth),
+        input=b2_conv0,
+        num_channels=384,
+        num_filters=256,
+        filter_size=(3, 1),
+        stride=1,
+        padding=(1, 0))
+    b3_conv0 = img_conv(
+        name='inceptC{0}_branch3_conv0'.format(depth),
+        input=input,
+        num_channels=1536,
+        num_filters=384,
+        filter_size=1,
+        stride=1,
+        padding=0)
+    b3_conv1 = img_conv(
+        name='inceptC{0}_branch3_conv1'.format(depth),
+        input=b3_conv0,
+        num_channels=384,
+        num_filters=448,
+        filter_size=(1, 3),
+        stride=1,
+        padding=(0, 1))
+    b3_conv2 = img_conv(
+        name='inceptC{0}_branch3_conv2'.format(depth),
+        input=b3_conv1,
+        num_channels=448,
+        num_filters=512,
+        filter_size=(3, 1),
+        stride=1,
+        padding=(1, 0))
+    b3_conv3 = img_conv(
+        name='inceptC{0}_branch3_conv3'.format(depth),
+        input=b3_conv2,
+        num_channels=512,
+        num_filters=256,
+        filter_size=(3, 1),
+        stride=1,
+        padding=(1, 0))
+    b3_conv4 = img_conv(
+        name='inceptC{0}_branch3_conv4'.format(depth),
+        input=b3_conv2,
+        num_channels=512,
+        num_filters=256,
+        filter_size=(1, 3),
+        stride=1,
+        padding=(0, 1))
+    return paddle.layer.concat(
+        input=[b0_conv0, b1_conv0, b2_conv1, b2_conv2, b3_conv3, b3_conv4])
+
+
+def Reduction_A(input):
+    b0_pool0 = paddle.layer.img_pool(
+        name='ReductA_branch0_pool0',
+        input=input,
+        num_channels=384,
+        pool_size=3,
+        stride=2,
+        pool_type=paddle.pooling.Max())
+    b1_conv0 = img_conv(
+        name='ReductA_branch1_conv0',
+        input=input,
+        num_channels=384,
+        num_filters=384,
+        filter_size=3,
+        stride=2,
+        padding=1)
+    b2_conv0 = img_conv(
+        name='ReductA_branch2_conv0',
+        input=input,
+        num_channels=384,
+        num_filters=192,
+        filter_size=1,
+        stride=1,
+        padding=0)
+    b2_conv1 = img_conv(
+        name='ReductA_branch2_conv1',
+        input=b2_conv0,
+        num_channels=192,
+        num_filters=224,
+        filter_size=3,
+        stride=1,
+        padding=1)
+    b2_conv2 = img_conv(
+        name='ReductA_branch2_conv2',
+        input=b2_conv1,
+        num_channels=224,
+        num_filters=256,
+        filter_size=3,
+        stride=2,
+        padding=1)
+    return paddle.layer.concat(input=[b0_pool0, b1_conv0, b2_conv2])
+
+
+def Reduction_B(input):
```

Comment:
The network structure here has some diffs in with the TF implementation of `Reduction_B `.

Reply:
Same as above.